packages/salesforcedx-vscode-core/src/commands/util/preconditionCheckers.ts

```diff
@@ -4,7 +4,7 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode/out/src/helpers';
+import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode';
```

Comment:
same comment - isNullOrUndefined seems generic and should be in salesforcedx-utils